backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+= Introduction to OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+
+The {oadp-first} product is intended to safeguard customer applications on the OpenShift Platform. OADP provides disaster recovery protection for OpenShift applications, application related cluster resources, persistent volumes, and internal images.
+
+OADP can be used to back up containerized applications and virtual machines (VMs).
```

Comment:
might be confusing
vm is also the short-hand name of the VirtualMachine Resouce type 
When br VM, it is not only the vm resource

Reply:
but it says *virtual machine* in front of it so there is no confusion